audit/formatter.go

```diff
@@ -21,6 +22,7 @@ type FormatterConfig struct {
 	Raw          bool
 	Salt         *salt.Salt
 	HMACAccessor bool
+	Mode         os.FileMode
```

Comment:
I didn't realize before that this was in `FormatterConfig`. It's not a property of the audit log format so needs to be a member of the file audit object, as that's the only place it has relevance.